Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -1902,6 +1902,50 @@ theorem insertRight_insertLeft {x x' x'' : PGame} :
   cases x; cases x'; cases x''
   dsimp [insertLeft, insertRight]
 
+/-! ### Removing an option -/
+
+/-- The PGame constructed by removing `'x` as a left option from `x`. -/
+def removeLeft (x x' : PGame.{u}) : PGame :=
+  match x with
+  | mk _ xr xL xR => mk { x // ¬(xL x ≡ x') } xr (fun i ↦ xL (↑i)) xR
```

Comment:
Instead of `x : PGame`, I think you should take `i : x.LeftMoves` as an argument.

Reply:
Interesting! I was expecting it to make matters worse, but it helped remove dealing with HEq!